R/rocco.R

```diff
@@ -10,12 +10,15 @@
 #'    immediately for browsing. This will be set to \code{\link{interactive}()},
 #'    that is, TRUE if the R session is running interactive and FALSE
 #'    otherwise.
+#' @param rocco logical. Whether or not to create rocco docs.  Defaults to \code{TRUE}.
+#' @param staticdocs logical. Whether or not to create staticdocs.  Staticdocs are
```

Comment:
You should be able to use `\href` instead of the ticks. http://r-pkgs.had.co.nz/man.html